ban-cooking.lua

```diff
@@ -2,6 +2,33 @@
 -- based on ban-cooking.rb by Putnam: https://github.com/DFHack/scripts/pull/427/files
 -- Putnams work completed by TBSTeun
 
+local options = {}
+
+local argparse = require('argparse')
+local commands = argparse.processArgsGetopt({...}, {
+    {'h', 'help', handler=function() options.help = true end},
+    {'v', 'verbose', handler=function() options.verbose = true end},
+})
+
+if options.help == true then
+    print(dfhack.script_help())
+    return
+end
+
+if commands.contains('all') then
+    for _, v in pairs(funcs) do
+        if _ ~= 'show' then
```

Comment:
ha, I didn't catch this in the original review. `_` is normally used for variables that you *don't* reference. this should have a name.

Reply:
we did wonder! =p